### ISSUE TITLE
Multidevice support & read frequency

### ISSUE THREAD

**Issue by Marquo1** [golles/ha-kamstrup_403#14]
### Did you read the instructions?

- [X] I have read the [README](../blob/master/README.md).

### The request

Firstly, I would also like to report a new device working with the integration. In my case it is Kamstrup Multical 601. Thank you for this integration!

Secondly, can you please specify what's the read frequency and ideally make it a configurable value? 

### Additional information

From my research, I understand that reading the meter drains its battery (~15 min of lifetime reduction per read with brand new battery lasting 10 years without readouts). It'd be good to have control over it if ones device is battery powered.

**Comment by golles** [comment]
Hi @Marquo1 

Thanks for reporting that this component works on the 601, I'll update the docs

The current update interval is 60 seconds:
https://github.com/golles/ha-kamstrup_403/blob/main/custom_components/kamstrup_403/__init__.py#L29

I can imagine that this is not suitable for battery-operated devices.
I'll make this configurable in the future

**Comment by golles** [comment]
Hi @Marquo1 I've made the change you suggested in #15 
Would it be possible to help test this change? You can do this by checking out the `configure_scan_interval` branch.

I'm testing this now as well, if all is fine I'll release this later this weekend.

**Comment by Marquo1** [comment]
@golles Splendid!

I'll try it in the morning and come back to you

**Comment by Marquo1** [comment]
Hi @golles 

Everything is working but it was hard to confirm if the scan interval was changed. It seems the entities only show they're updated when the value has changed. Timing the updates with a stopwatch while my SO is showering doesn't feel creepy at all 🤣

**Comment by golles** [comment]
I noticed an issue that I've fixed 2 hours ago.

I've tested it by changing the value to 10 seconds and I could see updates every 10 seconds. I had my logger set as:
```yaml
logger:
  default: warn
  logs:
    custom_components.kamstrup_403: debug
```

**Comment by golles** [comment]
I've merged the PR and released `1.2.0` https://github.com/golles/ha-kamstrup_403/releases/tag/1.2.0

Thank you for the suggestion for this option and confirming that it works on the 601 meter.

Please star this repository if you're happy with this component - it makes me happy :-)

**Comment by Marquo1** [comment]
> I've tested it by changing the value to 10 seconds and I could see updates every 10 seconds. I had my logger set as:
> 
> ```yaml
> logger:
>   default: warn
>   logs:
>     custom_components.kamstrup_403: debug
> ```

Ah that's experience I don't have. ;) Thanks for listening!
